lib/smoke_tests/gateway/user_details.rb

```diff
@@ -0,0 +1,26 @@
+require "logger"
+
+class SmokeTests::Gateway::UserDetails
+  SESSION_BATCH_SIZE = 500
+  def delete_users
+    logger = Logger.new(STDOUT)
+    logger.info("Starting daily smoke test user deletion")
+
+    total = 0
+    loop do
+      deleted_rows = DB[:userdetails].with_sql_delete("
```

Comment:
could we use the Sequel interface here and the ```user``` model defined in this repo?

Reply:
We could... this was created as a clone of https://github.com/alphagov/govwifi-user-signup-api/blob/master/lib/gdpr/gateway/usedetails.rb so mainly replicates the structure there - same for some other points.